libsql-client/examples/select.rs

```diff
@@ -70,7 +73,16 @@ async fn bump_counter(db: impl Connection) -> String {
 
 #[tokio::main]
 async fn main() {
-    let db = libsql_client::reqwest::Connection::connect_from_env().unwrap();
-    let response = bump_counter(db).await;
-    println!("{response}")
+    match libsql_client::reqwest::Connection::connect_from_env() {
+        Ok(remote_db) => {
+            let response = bump_counter(remote_db).await;
+            println!("Remote:\n{response}");
+        }
+        Err(e) => println!("Failed to fetch from a remote database: {e}"),
+    }
+
+    let local_db =
+        libsql_client::local::Connection::connect("/tmp/libsql_client_test_db.db").unwrap();
```

Comment:
Would it make sense to use some randomised file name to avoid reusing the same db file? With reusing we might get unexpected results and sometimes see correct answer even if the code gets broken.

Reply:
In this particular case I want it reused - then the counter in this example can actually be bumped multiple times by running the example more than once. Later we should definitely add proper tests to `libsql-client`, and then I'll definitely go with randomized names and seeds and stuff

Reply:
Makes sense.